File: scratch/parse.js
const fs = require('fs');
const css = fs.readFileSync('css/styles.css', 'utf-8');
const lines = css.split('\n');
lines.forEach((line, i) => {
    if (line.toLowerCase().includes('overflow')) {
        console.log(`Line ${i+1}: ${line.trim()}`);
    }
});

File: scratch/parse.py
import re

with open("c:\\Users\\dpradel\\OneDrive - Nelogica\\Documentos\\Tryd\\css\\styles.css", "r", encoding="utf-8") as f:
    css = f.read()

lines = css.splitlines()
for i, line in enumerate(lines):
    if "overflow" in line.lower():
        print(f"Line {i+1}: {line.strip()}")
